webapp/channels/src/components/integrations/abstract_command.tsx

```diff
@@ -436,15 +430,16 @@ export default class AbstractCommand extends React.PureComponent {
                                         defaultMessage='Reserved: {link}'
                                         values={{
                                             link: (
-                                                <ExternalLink
+                                                <a
                                                     href='https://developers.mattermost.com/integrate/admin-guide/admin-slash-commands/#built-in-commands'
-                                                    location='abstract_command'
+                                                    target='_blank'
```

Comment:
style checker is complaining about it.

Reply:
oh sorry, didn't check this, I was testing something

Reply:
made the changes